From e929c07b1e7a8c4b52e5efa11cc664f4f41c0a78 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 9 Jul 2018 19:19:00 -0400
Subject: [PATCH] joyent/node-assert-plus#32 package.json repository does not
 point to Joyent Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
 Approved by: Patrick Mooney <patrick.mooney@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 40d6a5c..9cb0e2d 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/mcavage/node-assert-plus.git"
+    "url": "https://github.com/joyent/node-assert-plus.git"
   },
   "engines": {
     "node": ">=0.8"
-- 
2.21.0

